src/webnn/native/dmlx/GraphDMLX.cpp

```diff
@@ -1270,7 +1270,7 @@ namespace webnn::native::dmlx {
         DAWN_ASSERT(mExpression.find(inputOperand) != mExpression.end());
         ::dml::Expression input = mExpression.at(inputOperand);
         auto newShape = reshape->GetNewShape();
-        if (newShape.size() > DML_TENSOR_DIMENSION_COUNT_MAX) {
+        if (newShape.size() > DML_TENSOR_DIMENSION_COUNT_MAX1) {
```

Comment:
@mingmingtasd Please update it to dml backend.

Reply:
OK, got it, will do it when re-writing DML backend. @fujunwei @miaobin

Reply:
Ack.